ui/StatusQ/src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -157,12 +157,12 @@ Item {
         \qmlproperty real StatusBaseInput::leftPadding
         This property sets the leftComponentLoader's left padding.
     */
-    property real leftPadding: leftComponentLoader.item ? 6 : 16
+    property real leftPadding: leftComponentLoader.status === Loader.Ready && leftComponentLoader.item ? 6 : 16
```

Comment:
Isn't `leftComponentLoader.status === Loader.Ready` redundant ?

Reply:
Yeah, it's strange but when testing this, it wouldn't work without